Package.swift

```diff
@@ -31,7 +35,10 @@ let package = Package(
         // Products define the executables and libraries a package produces, making them visible to other packages.
         .library(
             name: "MCP",
-            targets: ["MCP"])
+            targets: ["MCP"]),
+        .library(
+            name: "MCPToolMacrosPlugin",
```

Comment:
Naming: The library product is `MCPToolMacrosPlugin`, and the compiler plugin type is also named `MCPToolMacrosPlugin` (in `Sources/MCPToolMacros/Plugin.swift`). Consider renaming the type to `MCPToolMacrosCompilerPlugin` to avoid confusion.